### PR TITLE
update file name

### PR DESCRIPTION
**Pull request by ElizabethSamuel-MSFT**

### CODE REVIEW

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/includes/file-get-started-powerpoint.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-update-file-name/docs/includes/file-get-started-powerpoint.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C7%5C23%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201807231856143765-259%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=%2FbXDHImJmcEKzx2vpHx1H9bLgV%2FVuZeJloXm77RJ2lA%3D&st=2018-07-23T18%3A52%3A18Z&se=2018-08-23T18%3A57%3A18Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/includes/file-get-started-powerpoint.md [hunk 1]
```diff
@@ -230,7 +230,7 @@ In this article, you'll walk through the process of building a PowerPoint add-in
     </div>
     ```
 
-4. Open the file **app.js** in the root of the project to specify the script for the add-in. Replace the entire contents with the following code and save the file.
+4. Open the file **index.js** in the root of the project to specify the script for the add-in. Replace the entire contents with the following code and save the file.
```
Comment (AlexJerabek) [inline docs/includes/file-get-started-powerpoint.md]: I just changed this for OneNote. Is this in the root folder or src/index.js?

Comment (ElizabethSamuel-MSFT) [reply docs/includes/file-get-started-powerpoint.md]: under src. I'll update it. Thx.

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/includes/file-get-started-onenote.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-update-file-name/docs/includes/file-get-started-onenote.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-powerpoint.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-update-file-name/docs/includes/file-get-started-powerpoint.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C7%5C23%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201807231907164369-259%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=h2rBePBd0v57%2BOtsJIuhDV40faZpRZOw3IJwVfFVLtc%3D&st=2018-07-23T19%3A05%3A12Z&se=2018-08-23T19%3A10%3A12Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**